types/ethjs-signer/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for ethjs-signer 0.1
+// Project: https://github.com/ethjs/ethjs-signer/
+// Definitions by: Bryson Thill <https://github.com/doppio>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+/// <reference types="node" />
+
+/**
+ * Signs a raw transaction and return it either as a serlized hex string or raw tx object.
+ */
+export function sign(transaction: object, privateKey: string, toObject?: boolean): string | any[];
```

Comment:
It sounds like this function returns a different type depending on the value of `toObject`. Could you use two separate overloads?
```ts
export function sign(..., toObject?: false): A;
export function sign(..., toObject: true): B;
```